packages/core/framework/src/build-tools/compiler.ts

```diff
@@ -472,54 +451,68 @@ export class Compiler {
    * The "onFileChange" argument can be used to get notified when
    * a file has changed.
    */
-  async developPluginBackend(onFileChange?: () => void) {
-    await this.#ensureDir(this.#pluginsDistFolder, true)
-    await this.#createPluginOptionsFile()
-    const ts = await this.#loadTSCompiler()
+  async developPluginBackend(
+    transformer: (filePath: string) => Promise<string>,
+    onFileChange?: () => void
```

Comment:
should this now pass the "file" and the "action" as arguments?

Reply:
Yeah, we can do that. But this `onFileChange` action for now doesn't need any info. But it won't hurt to provide this info